CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* [#7676](https://github.com/rubocop-hq/rubocop/pull/7676): Update documentation to reflect current configuration precedence of TargetRuby. ([@drewcimino][])
```

Comment:
Documentation changes don't need a changelog entry.

Reply:
Got it 👍